docs/Configuration.md

```diff
@@ -204,9 +204,10 @@ language from being allowed. If there are more than one `locale tag`
 (with different country codes) for the same language, then
 all those must be removed to block that language.
 
-English is the Zonemaster default language, but it can be blocked
-from being allowed by RPC-API by not including it in the
-configuration.
```

Comment:
This section should not be removed. It is updated in another PR in commit https://github.com/zonemaster/zonemaster-backend/pull/796/commits/b3a51cb1e683747f2a0e72ba75ba8711bdd41f1f

Now that translation is added for error messages, we still have the translation of test result, and that should not change.

Reply:
I've re-added it.